### ISSUE TITLE
noDuplicateProperties doesn't allow fallback CSS [beta]

### ISSUE THREAD

**Issue by hyperknot** [biomejs/biome#6051]
### Environment information

```bash
Playground repros
```

### Rule name

lint/suspicious/noDuplicateProperties

### Playground link

https://next.biomejs.dev/playground/?code=LgBQAHIAbwBzAGUATQBpAHIAcgBvAHIAIAB7AAoAIAAgAHcAaABpAHQAZQAtAHMAcABhAGMAZQA6ACAAcAByAGUALQB3AHIAYQBwADsACgAgACAAdwBoAGkAdABlAC0AcwBwAGEAYwBlADoAIABiAHIAZQBhAGsALQBzAHAAYQBjAGUAcwA7AAoAfQA%3D&language=css&ruleDomains.solid=recommended

### Expected result

In CSS the only way to specify fallback properties for older browsers is to have the same property twice.
lint/suspicious/noDuplicateProperties should allow these on CSS.  

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
I think it's fair to disable or suppress the rule then. When targeting modern browsers, duplicated properties are not expected in most cases.

**Comment by hyperknot** [comment]
The point of this use case is to target modern browsers but provide a fallback to older ones. It's a recommended practice in CSS.

**Comment by ematipico** [comment]
Thank you for your feedback. As for now Biome wants to focus on modern best practices.

 In the future we could allow Biome to configure the target browser
